_includes/pbc/all/install-features/202212.0/install-the-agent-assist-feature.md

```diff
@@ -366,6 +368,7 @@ class SecurityDependencyProvider extends SprykerSecurityDependencyProvider
     protected function getSecurityPlugins(): array
     {
         return [
+            ...
```

Comment:
I do not think we do this is our documentation. The code in the snippet should be fully valid without any cleanups. Please remove the `...`.

Reply:
@limeeugenia - removed.